### DISCUSSION TITLE
Fablab website

### DISCUSSION THREAD

**Discussion by jollydodo** [thuliteio/doks#1187] [Showcase]
Hey! 

Our website is finally live. Thank you Henk for creating this great theme and helping me with all the (many) questions that I had! 
I chose this theme, because the goal of a Fablab should be to share knowledge about digital fabrication technologies with others so a wiki/docs was really something that was missing on our old website. However, I also added custom sections for contact/about/blog/workshops/gallery that fit our specific need while trying to stay as close to the original theme as possible (for flexibility purposes). 

Domain:
https://www.fablabarnhem.nl

Thanks again! (And off course feel free to visit us if you are in the neighbourhood sometime ;) )

**Comment by h-enk** [comment]
Thank you — and you're welcome! I'm very impressed with the result — super — I added the Fablab Arnhem website to the [showcase](https://getdoks.org/showcase/). Hope you don't mind  (I'll keep the invitation surely in mind) 😆
